README.rst

```diff
@@ -24,15 +24,15 @@ Description
 This is a tool for uploading certain kinds of files to CGAP.
```

Comment:
This makes sense. I committed a change for this, note that the links to readthedocs I added won't work yet because it's not in the master branch for readthedocs ("latest").